### DISCUSSION TITLE
Best Practice around company's tokens

### DISCUSSION THREAD

**Discussion by kefimochi** [vercel/vercel#5245] [Help] [answered]
It seems as though tokens used in CI/CD pipeline have to be generated on individual level rather than org's. 

I'd love to know what is the best practice around that(and if in the future company-level tokens would be supported) 🙂

**Comment by leerob** [comment, marked as answer]
https://vercel.com/changelog/access-tokens-can-now-be-scoped-to-teams

**Reply by enesse** [reply]
@leerob; But these are still tied to my account, aren't they? It's just a matter of scoping the token. So let say I leave the team and move on to a new project, the token stops working I assume? If that's the case, is there any way of creating a token that works independently of specific team members -- a team token?

**Reply by Jorundur** [reply]
To continue the tradition of this discussion receiving a reply every 2 years 😅, I'd like to add here that the only solution to this that I've seen is a special "team user". I.e. create a new personal account called "my-team-ci" or something and assign personal access tokens to that account.
